File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        breakpoints = 0
        for i in range(len(nums)-1):
            if nums[i]>nums[i+1]:
                breakpoints += 1
        if nums[len(nums)-1]>nums[0]:
            breakpoints +=1
        if breakpoints==1 or breakpoints==0:
            return True
        else:
            return False
        
                

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int breakpoints = 0;
        for(int i=1; i<nums.size();i++){
            if(nums[i]<nums[i-1]){
                breakpoints++;
            }
        }
        if(nums[nums.size()-1]>nums[0]){
            breakpoints++;
        }
        if(breakpoints==1 or breakpoints==0){
            return true;
        }
        else{
            return false;
        }
    }
};